torch/_inductor/ir.py

```diff
@@ -6030,6 +6031,16 @@ def codegen(self, wrapper):
         if isinstance(self.layout, Layout):
             self.codegen_size_asserts(wrapper)
 
+    def get_mutation_names(self):
```

Comment:
I think you also need to add
```
def get_unbacked_symbol_defs(self):
        return {}
```
otherwise it might use some parent class definition

Reply:
Thanks and added. BTW: @oulgen what's the meaning of `unbacked_symbol`?

Reply:
It is used for scheduler to track some symbol dependancies but for this op there are no unbacked symbols, so it is not needed